static/examples/cstg-prebid-example/index.html

```diff
@@ -9,67 +9,182 @@
+      
+      async function handleLoginEUID() {
+        const email = document.querySelector('#email_euid').value;
+        setConfigEUID(email);
+        await pbjs.refreshUserIds();
+        updateGuiElementsEUID();
+      }
+
+      function handleClearStorage(idType) {
         console.log('Clearing storage.');
-        localStorage.removeItem('__uid2_advertising_token');
-        location.reload();
+        localStorage.removeItem(`__${idType}_advertising_token`);
+        $(`#targeted_advertising_ready_${idType}`).text('no');
+        $(`#advertising_token_${idType}`).text('');
+        $(`#login_form_${idType}`).show();
+        $(`#clear_storage_form_${idType}`).hide();
+      }
+
+      function handleClearStorageUID2() {
+        handleClearStorage("uid2");
+      }
+
+      function handleClearStorageEUID() {
+        handleClearStorage("euid");
       }
 
+
       function onDocumentReady() {
         console.log('Setting up interface handlers.');
-        $('#login').click(handleLogin);
-        $('#clear_storage').click(handleClearStorage);
+        $('#login_uid2').click(handleLoginUID2);
+        $('#clear_storage_uid2').click(handleClearStorageUID2);
+        $('#login_euid').click(handleLoginEUID);
+        $('#clear_storage_euid').click(handleClearStorageEUID);
         
-        updateGuiElements();
+        updateGuiElementsUID2();
+        updateGuiElementsEUID();
+      }
+
+      let consentManagementObj = {
+        "cmpApi": "static",
+        "consentData": {
+          "getTCData": {
+            "tcString": "CO-HDlqO-HDlqAKAXCENBDCsAP_AAH_AACiQHKNd_X_fb39j-_59_9t0eY1f9_7_v20zjgeds-8Nyd_X_L8X42M7vF36pq4KuR4Eu3LBIQFlHOHcTUmw6IkVqTPsak2Mr7NKJ7PEinMbe2dYGHtfn9VTuZKYr97s___z__-__v__75f_r-3_3_vp9V---_fA5QAkw1L4CLMSxwJJo0qhRAhCuJDoAQAUUIwtE1hASuCnZXAR-ggYAIDUBGBECDEFGLIIAAAAAkoiAkAPBAIgCIBAACAFSAhAARoAgsAJAwCAAUA0LACKAIQJCDI4KjlMCAiRaKCeSMASi72MMIQyigBoFH4AAAAA.cAAAAAAAAAAA",
+            "cmpId": 10,
+            "cmpVersion": 23,
+            "tcfPolicyVersion": 2,
+            "gdprApplies": true,
+            "cmpStatus": "loaded",
+            "eventStatus": "tcloaded",
+            "purpose": {
+              "consents": {
+                "1": true,
+                "2": true
+              }
+            },
+            "vendor": {
+              "consents": {
+                // add your GVL ID here and set to true to give consent within pbjs
+                "52": true,     // rubicon for adserving
+                "21": true,     // unifiedId
+                "131": true,    // id5Id
+                "929": true,    // parrableId
+                "97": true,     // identityLink
+                "887": true,    // uid2
+                "95": true,     // lotamePanoramaId
+                "301": true,    // zeotapIdPlus
+                "91": true,     // criteo
+                "737": true,    // amxId
+                "58": true,     // 33acrossId
+              }
+            }
+          }
+        }
```

Comment:
Design decision here - for UID2, technically all that needs to be sent is "consentData" -> "getTCData" -> "gdprApplies" but if `consentData` does not have some other properties, the code fails.  So to make it clear what is changing between UID2 and EUID with the "consentManagement" prop, I thought it best to keep the entire object defined and only change if "gdprApplies" is true or false based on the idType